.travis.yml

```diff
@@ -3,6 +3,7 @@ language: php
 php:
   - 5.6
   - 7.0
+  - 7.1.3
```

Comment:
Why not simply 7.1? Also we can take benefit to try 7.2 here.

Reply:
Agree, we can put 7.1, also 7.2 and i think we should delete the 5.6, and 7.0 because the symfony 4  min requirements for php are `^7.1.3`